apps/token-manager/app/src/screens/Holders.js

```diff
@@ -1,166 +1,163 @@
-import React from 'react'
+import React, { useCallback, useState } from 'react'
 import PropTypes from 'prop-types'
+import { useNetwork } from '@aragon/api-react'
 import styled from 'styled-components'
 import {
+  ContextMenu,
+  ContextMenuItem,
+  DataView,
+  IconAdd,
+  IconLabel,
+  IconRemove,
+  Split,
   TabBar,
   Table,
   TableHeader,
   TableRow,
   Viewport,
   breakpoint,
+  useTheme,
 } from '@aragon/ui'
-import HolderRow from '../components/HolderRow'
-import SideBar from '../components/SideBar'
+import { formatBalance } from '../utils'
+import InfoBoxes from '../components/InfoBoxes'
+import LocalIdentityBadge from '../components/LocalIdentityBadge/LocalIdentityBadge'
+import { useIdentity } from '../components/IdentityManager/IdentityManager'
 
-const TABS = ['Holders', 'Token Info']
+function Holders({
+  groupMode,
+  holders,
+  maxAccountTokens,
+  onAssignTokens,
+  onRemoveTokens,
+  tokenAddress,
+  tokenDecimalsBase,
+  tokenName,
+  tokenSupply,
+  tokenSymbol,
+  tokenTransfersEnabled,
+  userAccount,
+}) {
+  const network = useNetwork()
+  return (
+    <Split
```

Comment:
Is this the magical component that structures the 1/2 column layout when the viewport size changes? Nice! ✨ ✨

Reply:
It is indeed! :laughing: https://github.com/aragon/aragon-ui/blob/newstyle/src/components/Split/Split.js